src/commissaire_http/authentication/keystonetokenauth.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2016  Red Hat, Inc
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+OpenStack Keystone authentication plugin.
+"""
+
+import json
+import requests
+
+from commissaire_http.authentication import Authenticator
+
+
+class KeystoneToken(Authenticator):
+    """
+    Auth implementation using token method against OpenStack Keystone
+    """
+
+    def __init__(self, app, url=None, scope=None):
+        """
+        Checks the token given as the X-Auth-Token header
+        against the specified OpenStack Keystone instance
+        :param app: The WSGI application being wrapped with authenticaiton.
+        :type app: callable
+        :param url: The OpenStack Keystone endpoint used for Authentication
+        :type url: string
+        :returns
+        """
+        super(KeystoneToken, self).__init__(app)
```

Comment:
Python 3 no longer requires ``super`` to define inputs. No need to change. Just a reminder that ``super().__init__(app)`` would work here too.